custom/icds_reports/ucr/data_sources/dashboard/dashboard_growth_monitoring.json

```diff
@@ -393,6 +393,9 @@
       },
       {
         "column_ids": ["state_id", "muac_grading"]
```

Comment:
These previous indexes were implemented in https://github.com/dimagi/commcare-hq/commit/38c305c0290e. I don't think they're necessary anymore, but I'm not totally sure and haven't yet looked into that.